app/controllers/fields_controller.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 class FieldsController < ApplicationController
-  def index
-    @field = Field.new
+  def show
+    @field = Field.find params[:id]
```

Comment:
Dùng find_by đi anh

Reply:
1. Why using find_by instert of find  while you mark index to id column ??
![image](https://github.com/awesome-academy/oe60-naitei10-7-soccer-fields-management-system/assets/59260191/0b423a91-38a2-4eaf-8c3b-781088a97257)

2. I'm handle exception  with handle_error_and_redirect hepler, if using find_by, you have to handle nil case.

3. find_by too many the code than find